src/_pytest/fixtures.py

```diff
@@ -302,7 +302,7 @@ def fillfixtures(function):
 
 
 def get_direct_param_fixture_func(request):
-    return request.param
+    return request.param if hasattr(request, "param") else None
```

Comment:
that change makes the fixture helper incorrect - if no parameter is existing this one has to raise a failure - the tests shouldnt even run however

Reply:
Shouldn't this raise an error then, or perhaps `pytest.fail("no direct parameter")`??